modules/postgres/postgres.go

```diff
@@ -13,7 +13,7 @@ import (
 const (
 	defaultUser          = "postgres"
 	defaultPassword      = "postgres"
-	defaultPostgresImage = "docker.io/postgres:11-alpine"
+	defaultPostgresImage = "docker.io/postgres:16-alpine"
```

Comment:
Can you also update the docs to use this version? 🙏

Reply:
Where is the version referenced?
ctrl-f on the project for `postgres:11` didn't show anything. Didn't see anything for version at https://golang.testcontainers.org/modules/postgres/

Reply:
Oh sorry, I meant changing the docs (using postgres:15) to the one you added 🙏 . The code is extracted from the examples_test.go file.

Reply:
As a consequence, we are going to update the testcontainers.com examples too. Thanks for pointing this out!

Reply:
Sorry, it's Monday and early. Do you want the examples to use the default image or have the line from examples `testcontainers.WithImage("docker.io/postgres:15.2-alpine"),`  update to 16?